augur/tasks/github/facade_github/contributor_interfaceable/contributor_interface.py

```diff
@@ -132,7 +132,9 @@ def create_endpoint_from_commit_sha(logger,db,commit_sha, repo_id):
 
     # Else put into a more readable local var
     #session.logger.info(f"Result: {result}")
-    repo_path = result.repo_path.split("/")[1] + "/" + result.repo_name
+
+    split_git = result.repo_git.split('/')
+    repo_path = split_git[-2] + "/" + result.repo_name
 
```

Comment:
I think this is parsing out the org / owner name so the variable name should be org or owner for clarity.

Reply:
That makes sense